### ISSUE TITLE
check-tag-names doesn't recognize @internal

### ISSUE THREAD

**Issue by ilyub** [gajus/eslint-plugin-jsdoc#995]
IMPORTANT:
There is exactly the same issue 1 year ago:
https://github.com/gajus/eslint-plugin-jsdoc/issues/859
It says that the issue was fixed in 38.0.8.
And yet I see it with 40.0.0.

## Expected behavior

Should recognize this.
```ts
/**
 * @internal
 */
```

## Actual behavior

Reports error:
Invalid JSDoc tag name "internal".

## ESLint Config

```ts
"jsdoc/check-tag-names": "warn",
```

## ESLint sample

```ts
/**
 * @internal
 */
export interface OnMouseDown {}
```

## Environment

- Node version: 18.15.0
- ESLint version 8.33.0
- `eslint-plugin-jsdoc` version: 40.0.0

**Comment by brettz9** [comment]
It was fixed then. Note the test provided that requires that:

```js
settings: {
        jsdoc: {
          mode: 'typescript',
        },
      },
```

...be provided.
